### DISCUSSION TITLE
取消授权（取消当前设备的高级账户资格）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1575] [高级账户]
>  又称 **解除当前设备**，由于 **简悦 · 高级账户支持三台设备**，如果有多于三台设备的话，可以通过此功能来解绑某个设备的高级账户，位置：**简悦 → 选项页 → 右上角 → 账户 icon → 解除当前设备的授权**

[![Bqv8OA.png](https://s1.ax1x.com/2020/11/10/Bqv8OA.png)](https://imgchr.com/i/Bqv8OA)

操作视频
---

https://user-images.githubusercontent.com/81074/139789210-19b8318a-cb44-4399-98b2-f08b5a31e01f.mp4

注意
---

1. 取消后的设备，仍旧可以使用简悦，**仅无法使用简悦 · 高级账户**功能。
2. 取消后的设备，仍旧可以通过**手动同步配置文件的方式同步你的简悦数据**。

关联阅读
---

- [绑定高级账户](https://github.com/Kenshin/simpread/discussions/1573)

- [在新设备上登录](https://github.com/Kenshin/simpread/discussions/1574)

- [使用授权登录](https://github.com/Kenshin/simpread/discussions/1570)

- [取消授权](https://github.com/Kenshin/simpread/discussions/1575)

- [重新授权](https://github.com/Kenshin/simpread/discussions/1576)

- [找回丢失的 UID （高级账户资格）](https://github.com/Kenshin/simpread/discussions/1571)

- [找回 UID 后如何恢复高级账户资格](https://github.com/Kenshin/simpread/discussions/1958)

- [重新找到 License（兑换码）](https://github.com/Kenshin/simpread/discussions/2756)

- [安装和配置同步助手](https://github.com/Kenshin/simpread/discussions/2754)

**Comment by zhuxiucai** [comment]
可不可以增加一个feature：
默认3台，新登陆设备默认挤掉老的（活跃时间距目前时间最远）？
这样，就可以不用再手动取消授权。

场景是在购买新设备的时候，一般想不到手动取消授权。都是把老设备直接给重置了。

**Reply by zhuxiucai** [reply]
@Kenshin

**Reply by Kenshin** [reply]
嗨，可以考虑下。

不过，当某个设备不再活跃时（超过一周）自然会被踢掉的。其实年底的时候会考虑出多设备的功能。
